frontend/src/localization/de.json

```diff
@@ -333,7 +333,7 @@
     "headline": "Filter-Werte aus Liste einfügen"
   },
   "conceptListDetailsModal": {
-    "conceptTreeRoot": "Konzeptbaum",
+    "conceptTreeRoot": "Konzept",
```

Comment:
Or "Konzept-Wurzel"?

Reply:
I would rather stay with "Konzept", since "Wurzel" is again tree terminology ;-)